marginalia.el

```diff
@@ -730,16 +762,47 @@ These annotations are skipped for remote paths."
       (marginalia--fields ("*Remote*" :face 'marginalia-documentation))
     (when-let (attributes (file-attributes (substitute-in-file-name (marginalia--full-candidate cand)) 'string))
       (marginalia--fields
-       ((file-attribute-modes attributes) :face 'marginalia-file-modes)
-       ((format "%s:%s"
-                (file-attribute-user-id attributes)
-                (file-attribute-group-id attributes))
+       ((concat (unless (string= user-login-name (file-attribute-user-id attributes))
+                  (file-attribute-user-id attributes))
+                (let ((group (file-attribute-group-id attributes)))
+                  (if (numberp group)
+                      (unless (= (group-gid) group)
+                        (concat ":" (number-to-string group)))
+                    (unless (string= (group-name (group-gid)) group)
```

Comment:
here we also need only the integer comparison

Reply:
That's interesting because on my system I get both string and int groups from `file-attribute-group-id`.

Reply:
This is consistent with the documentation
```
This is either a string or a number.  If a string value cannot be
looked up, a numeric value, either an integer or a float, is
returned.
```

Reply:
As I read it, this depends on the ID-FORMAT passed to `file-attributes`. See my comment above.

Reply:
Changed.